registry/README.md

```diff
@@ -15,6 +15,8 @@ features, you need to Sign In to Netdata cloud. By signing in, you opt in to let
 information described [here](#what-data-does-the-registry-store). You can still get the node menu, but not the cloud
 features, if you [run your own registry](#run-your-own-registry).
 
+[This is a broken link](/broken/link/README.md).
+
```

Comment:
I _assume_ you will remove this later in this PR once reviewed/approved :)

Reply:
You read my mind :smile: